metadata.rb

```diff
@@ -1,8 +1,8 @@
-name             'chef-server-image'
+name             'chef_server_image'
 maintainer       'YOUR_COMPANY_NAME'

```

Comment:
We should change the company name to Chef Software, Inc. and remove the maintainer_email.